### DISCUSSION TITLE
How i split the number of columns with Vaex

### DISCUSSION THREAD

**Discussion by ghost** [vaexio/vaex#1871] [Q&A] [answered]
Hello guys ,

I ve got something like this:

![image](https://user-images.githubusercontent.com/39860343/151700021-36b38f6d-e413-42fa-ab13-26dc9f9c58d8.png)

How i can split my df in multiple columns? vaex_df[0]

I really like vaex and find your work inspiring : )
BR

**Comment by JovanVeljanoski** [comment, marked as answer]
If this is your starting point.. i would suggest handling the splitting before you start working with vaex. 
Or perhaps if you reading this from a csv file, there are parameters you can adjust so that you get columns right away. 

Maybe you can share a bit more about your workflow?

**Comment by ghost** [comment]
Hi Jovan,

Please dont mind that, i solved it just by playing/adjusting delimiters and manually editing the csv.

Thanks for your answer again.

Unfortunately, i ve got another problem now with my dataset. I will start a new discussion subject in the future.

Wish you the best.
J.
